.travis.yml

```diff
@@ -47,6 +47,18 @@ matrix:
         - cargo build --manifest-path futures-sink/Cargo.toml --all-features
         - cargo build --manifest-path futures-util/Cargo.toml --all-features
 
+    - name: cargo build --all-features (with minimal versions)
+      rust: nightly
+      script:
+        - cargo update -Zminimal-versions
+        - cargo build --manifest-path futures/Cargo.toml --all-features
+        - cargo build --manifest-path futures-core/Cargo.toml --all-features
+        - cargo build --manifest-path futures-channel/Cargo.toml --all-features
+        - cargo build --manifest-path futures-executor/Cargo.toml --all-features
+        - cargo build --manifest-path futures-io/Cargo.toml --all-features
+        - cargo build --manifest-path futures-sink/Cargo.toml --all-features
+        - cargo build --manifest-path futures-util/Cargo.toml --all-features
```

Comment:
Could we use `cargo build --all-features` or `cargo build -p futures-... --all-features` instead?

`cargo build --no-default-features` seems not to exist at least

Reply:
`cargo build --all-features` might also not work. It exists but I'm not sure whether it does what we want

Reply:
Nope, features and workspaces are pretty much broken at the moment. There was an attempt at fixing them a while ago, but I’m not sure if that’s still available even as an unstable flag. I’ll try and find the link to the issue in a sec.

Reply:
https://github.com/rust-lang/cargo/issues/5364

Reply:
I tested `-p` instead of `--manifest-path` and it doesn't work